In [2]:
numArray = range(1,1000000)

In [3]:
numArray

range(1, 1000000)

In [15]:
for i in range(10):
    print( i )

0
1
2
3
4
5
6
7
8
9


Какое значение будет на 2-м месте?

1

In [18]:
for i in range(10):
  print( i**2 )

0
1
4
9
16
25
36
49
64
81


In [19]:
weekDays = ['пн', 'вт','ср','чт','пт','сб','вс']

for day in weekDays:
    print('Сегодня', day)

Сегодня пн
Сегодня вт
Сегодня ср
Сегодня чт
Сегодня пт
Сегодня сб
Сегодня вс


In [21]:
minValue = 4
maxValue = 18

In [22]:
range(100, 200)

range(100, 200)

In [24]:
numlist = range(minValue, maxValue)

In [27]:
lst = []
for i in range(minValue, maxValue, 2):
  lst.append(i**(1/2))

In [28]:
lst

[2.0,
 2.449489742783178,
 2.8284271247461903,
 3.1622776601683795,
 3.4641016151377544,
 3.7416573867739413,
 4.0]

Какое значение будет предпоследним?

In [32]:
lst[len(lst)-2]

3.7416573867739413

In [33]:
directRevenue = [83171,151604, 46315, 98753, 208648, 184682,
204061,134911,94791,109076,37254, 224991,36400,149320, 171336, 83854,
206799,180922, 235647, 217546, 200478, 239445, 144901,
26522,177971,148458,154937,196095,140202,189223]

In [35]:
week_1_revenue = directRevenue[0:6]

In [39]:
total_revenue = 0
for day in week_1_revenue:
    total_revenue+=day
    print('текущее значение = {}; сумма = {}'.format(day, total_revenue))
total_revenue

текущее значение = 83171; сумма = 83171
текущее значение = 151604; сумма = 234775
текущее значение = 46315; сумма = 281090
текущее значение = 98753; сумма = 379843
текущее значение = 208648; сумма = 588491
текущее значение = 184682; сумма = 773173


773173

Какая промежуточная сумма на 4 шаге цикла?

текущее значение = 98753; сумма = 379843

In [41]:
queriesText = 'смотреть сериалы онлайн;новости спорта;афиша кино;курс доллара;сериалы этим летом;курс по питону;сериалы про спорт'

In [42]:
lst = queriesText.split(";")

In [43]:
total_words = 0
for sentence in lst:
    words = sentence.split(" ")
    total_words += len(words)

total_words
    

18

In [45]:
semantic_list = ['одеяло !купить', 'одеяло !продажа','одеяло цена', 'одеяло стоимость',
                 'одеяло прайс','одеяло дешево', 'одеяло недорого','одеяло заказать','одеяло на заказ', 
                 'одеяло с доставкой', 'одеяло магазин','одеяло интернет магазин', 'одеяло со скидкой',
                 'одеяло акция','одеяло распродажа']

string_to_find = 'одеяло по дешевке'
if string_to_find in semantic_list:
    print('"{}" есть в списке'.format(string_to_find))
else:
    print('"{}" нет в списке'.format(string_to_find))


"одеяло по дешевке" нет в списке


In [46]:
text = 'Название языка питон произошло вовсе не от вида пресмыкающихся. Автор назвал язык в честь популярного британского комедийного телешоу 1970-х Летающий цирк Монти Пайтона'
name_to_find = 'Монти'

if name_to_find in text:
    print('В этом тексте есть упоминание "{}"'.format(name_to_find)) 

В этом тексте есть упоминание "Монти"


## Домашнее задание

In [1]:
queries = "смотреть сериалы онлайн,новости спорта,афиша кино,курс доллара,сериалы этим летом,курс по питону,сериалы про спорт"

In [2]:
words = ['сериалы', 'курс']

In [5]:
lst_result = []
lst_queries = queries.split(",")

In [8]:
for query in lst_queries:
    for word in words:
        if word in query:
            lst_result.append(query)

lst_result
    

['смотреть сериалы онлайн',
 'курс доллара',
 'сериалы этим летом',
 'курс по питону',
 'сериалы про спорт']

## Дополнительные задачи (продвинутый уровень).

#### 1

In [6]:
res = 0
inp = -1
while inp != 0:
    inp = int(input())
    print("Введено значение: ", inp)
    res += inp
print("Итоговый результат: ", res)

2
Введено значение:  2
3
Введено значение:  3
5
Введено значение:  5
3
Введено значение:  3
6
Введено значение:  6
0
Введено значение:  0
Итоговый результат:  19


#### 2

In [10]:
while 'true':
    inp = int(input())
    if inp > 100:
        break
    elif inp > 10:
        print("Введено значение: ", inp)

0
1
2
3
56
Введено значение:  56
67
Введено значение:  67
11
Введено значение:  11
10
99
Введено значение:  99
100
Введено значение:  100
101


#### 3

In [7]:
str1 = '1,23,4,25,6,7,7,4,53,4,6,3'
str2 = '5'
lst_num = str1.split(',')

idx = []
i = 0
for elem in lst_num:
    if elem == str2:   
        idx.append(i)
    i = i + 1

if len(idx) == 0:
    print('Отсутсвует')
else:
    print(idx)


Отсутсвует
